app/controllers/system_messages_controller.rb

```diff
@@ -15,20 +17,28 @@ def create
     @system_message.user = current_user
 
     if @system_message.save
-      @system_message.send!
+      @system_message.send!(current_user)
       redirect_to "/rad_common/system_messages/#{@system_message.id}", notice: 'The message was successfully sent.'
     else
       render :new
     end
   end
 
+  def twilio
```

Comment:
Is this needed? It may just be used for robo calls?

Reply:
removed